publications-2018.bib

```diff
@@ -723,6 +723,29 @@ @article{GKK18
    doi     = {}
 }
 
+@PhdThesis{SergeyCharnyiPhd2018,
+  Title                    = {The {EMAC} scheme for Navier-Stokes simulations, and application to flow past bluff bodies},
+  Author                   = {Sergey Charnyi},
+  School                   = {Clemson University},
+  Year                     = {2018},
+  Url                      = {https://tigerprints.clemson.edu/all_dissertations/2243/}
+}
+
+@PhdThesis{LiangZhaoPhd2018,
+  Title                    = {A numerical method for the Navier-Stokes equations in velocity-vorticity form},
```

Comment:
It doesn't really bother me, but the title of this thesis has different capitalisation on the linked website:
> A Numerical Method for the Navier Stokes Equations in a Velocity-Vorticity Form